### DISCUSSION TITLE
Particulate Matter by County

### DISCUSSION THREAD

**Discussion by LiamGearyUW** [jakobzhao/geog495#48] [Final Project]
**Link:** https://liamgearyuw.github.io/geog495_final/
**GitHub:** https://github.com/LiamGearyUW/geog495_final
**Description:** This project is an interactive web map featuring the counties of the 50 United States and Puerto Rico, as well as data on the levels of particulate matter in each county where such data is applicable. Particulate matter is a form of pollution that includes fine inhalable particles. Such pollution can enter the lungs and bloodstream, causing health risks.
![image](https://user-images.githubusercontent.com/91765139/145308250-0a3f7ba0-a375-46fb-8cd4-d8ca67e23125.png)

**Comment by VaneRaklan** [comment]
I'm not sure if it was what you were going for but I like the minimalist approach! As much as it would be nice to have a more complete data set for the mass amount of counties that are in the grey, and perhaps we can learn something from that lack of data, I think that the sparsity lends itself to your choropleth map because it really makes the color scale across the counties stand out! 

I'm kind of unsure as to what to critique. At the very least, one of my initial impressions when opening up your application was that I wanted more interaction. To a lesser degree, maybe highlighting the outline of the county that the cursor is hovering over. To a higher degree, maybe a click event of some sort that would populate a tooltip or section of the side panel with some information pertaining to what their particulate matter concentration means (i.e. for the people living there).

**Comment by zhuxiaoyu1019** [comment]
By graying down the target area and integrating it with the county layer, you clearly indicate the county boundaries. You also make a good choice on the color scheme for indicating the level of pollution. A lighter color would mean better air quality than a darker color without adding any additional information.
Maybe better data can be considered for this map since almost half of the county has no data, which makes this map a little meaningless. Are there any particular reasons why you split your legend evenly into 7 categories rather than more or fewer subcategories?

**Comment by jacobf19** [comment]
This is an interesting idea and an important topic to be shown. I like how appealing the map is to look at and the colors and style serve this purpose well. I would suggest trying to find a bit more data to fill out the rest of the country which has no data currently. Especially for a state like California, with Southern California being notorious for smog issues, it seems a bit strange not to have any data for the state which may be the most interesting to look at for users.

**Comment by kevography** [comment]
I think my favorite aspect of this map is how responsive the sidebar element is when hovering over a county. If I had to make a suggestion, it would be that there is a lot of empty space in the sidebar - I recommend defining the height to an appropriate size.

**Comment by manhongg** [comment]
Your sidebar was very informative and you did a thorough job of sorting through the dataset and presenting the data in a way that's user-friendly and informative. I liked that you chose to convey the information using a choropleth, as I felt it was a unique approach to this project and worked very well with the information you were sharing. I feel like adding more interactivity would improve your project and allow the user to engage with the data on a deeper level. Great job on your project!

**Comment by nlertv** [comment]
My favourite part of the map would be the sidebar; it contains a lot of helpful information in a way that is not too clustering. I also like how you include where you get the data from as well. One suggestion would be to find more data on the cities that are currently grey. I think this will make the map looks even more appealing.
